src/UI/molecules/DashboardElement.tsx

```diff
@@ -25,6 +26,14 @@ function DashboardElement({
   order = 0,
   title
 }: DashboardElementProps): JSX.Element {
+  const descriptionRef = useRef<HTMLDivElement>(null);
+  const [childrenHeight, setChildrenHeight] = useState<string>(null);
+
+  useLayoutEffect(() => {
+    const height: string = calcChildrenHeight();
+    setChildrenHeight(height);
+  });
```

Comment:
There's a warning in the console:
```
./src/UI/molecules/DashboardElement.tsx
  Line 32:3:  React Hook useLayoutEffect contains a call to 'setChildrenHeight'. Without a list of dependencies, this can lead to an infinite chain of updates. To fix this, pass [] as a second argument to the useLayoutEffect Hook  react-hooks/exhaustive-deps
```

```suggestion
  const calcChildrenHeight = useCallback((): string => {
    if (descriptionRef.current) {
      const { height }: DOMRect = descriptionRef.current.getBoundingClientRect();

      return `calc(100% - ${spacing.spacing36} - ${spacing.spacing28} - ${height}px)`;
    } else {
      return `calc(100% - ${spacing.spacing36})`;
    }
  }, [descriptionRef.current]);

  useLayoutEffect(() => {
    const height: string = calcChildrenHeight();
    
    setChildrenHeight(height);
  }, [calcChildrenHeight]);
```